modules/openxr/action_map/openxr_interaction_profile_metadata.h

```diff
@@ -64,21 +64,21 @@ class OpenXRInteractionProfileMetadata : public Object {
 	struct TopLevelPath {
 		String display_name; // User friendly display name (i.e. Left controller)
 		String openxr_path; // Path in OpenXR (i.e. /user/hand/left)
-		String openxr_extension_name; // If set, only available if extension is enabled (i.e. XR_HTCX_vive_tracker_interaction)
+		String openxr_extension_names; // If set, only available if extension is enabled (i.e. XR_HTCX_vive_tracker_interaction)
```

Comment:
Since we're updating the api, should we make this (and the ones below) a `List<String>` instead of a `String`?

Reply:
Can you add a `TODO` to track.